### PR TITLE
Adds ibmMQ example

### PR DESCRIPTION
**Pull request by githubsands**

### CODE REVIEW

File: collector/ibmmq/config.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+          metrics_path: "/metrics"
+          static_configs:
+            - targets: [":9157"]
+exporters:
+  logging:
+    logLevel: debug
+  otlp:
+    endpoint: ingest.lightstep.com:443
+    headers:
+      - lightstep-access-token: "eyJhbGciOiJIUzI1NiIsImtpZCI6IjIwMTktMDMtMDEiLCJ0eXAiOiJKV1QifQ.eyJzY3AiOnsicm9sZSI6ImU1MTI5ODNiLTFjYjktMTFlOC05M2Y1LTQyMDEwYWYwMGFkNiJ9LCJ2ZXIiOjEsImRlYnVnIjp7Im9yZyI6IkxpZ2h0U3RlcCIsInJvbGUiOiJPcmdhbml6YXRpb24gQWRtaW4ifSwiYXVkIjoiYXBwLmxpZ2h0c3RlcC5jb20iLCJleHAiOjE2ODUwMzc0ODcsImp0aSI6InlhM25uemVoenlyaW1jZnpuM29ya3AyMnh2cDNqY2JuY3F3anpwaHprNTJ3bmdubCIsImlhdCI6MTY1MzUwMTQ4NywiaXNzIjoibGlnaHRzdGVwLmNvbSIsInN1YiI6IjEyOWY2ZWIzNGE3ZGJjZDcxZTI2NDM1Yzc4OSJ9.rjKjrFkO5nXJCKfdMiPnfUSgO8FeREULILpnEO1_Cko="
```
Comment (jmacd) [inline collector/ibmmq/config.yaml:18]: I think we should leave this sort of token out of the repository. Is there a recommended solution for this in the other examples?

Comment (carlosalberto) [reply collector/ibmmq/config.yaml:18]: We indeed do not show it, see https://github.com/lightstep/opentelemetry-examples/pull/102/files#diff-2806d525ff2fef4464988f7d5a3deca463163c46a0a1c017ae9e43bbffc02a62R14

Comment (carlosalberto) [reply collector/ibmmq/config.yaml:18]: Ping @githubsands

**Comment by carlosalberto** [comment]
@githubsands I don't see metrics being reported upon running this example, and I also don't see any mention of the collector part if docker-compose, maybe that's missing?

**Comment by githubsands** [comment]
> @githubsands I don't see metrics being reported upon running this example, and I also don't see any mention of the collector part if docker-compose, maybe that's missing?

_resolved in slack_

File: collector/ibmmq/docker-compose.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+      - LICENSE=accept
+      - MQ_APP_PASSWORD=password
+      - MQ_ENABLE_METRICS=true
+    volumes:
+      - ./mqs.ini:/var/mqm/mqm
+    ports:
+      - 9157:9157
+    networks:
+      - localhost
+
```
Comment (codeboten) [inline collector/ibmmq/docker-compose.yaml:18]: @githubsands should there be an otel collector defined here somewhere?

**Pull request merged**